packages/dynamic_layouts/.gitignore

```diff
@@ -0,0 +1,30 @@
+# Miscellaneous
```

Comment:
Can we just use the global `.gitignore` [1] or do we need a special one?

[1] https://github.com/flutter/packages/blob/main/.gitignore

Reply:
Oh I was looking at adaptive_scaffold for comparison, it has the same `.gitignore`, should they be removed?